lib/jekyll-mentions.rb

```diff
@@ -31,7 +31,7 @@ def mentionify(doc)
       def filter_with_mention(src)
         filters[src] ||= HTML::Pipeline.new([
           HTML::Pipeline::MentionFilter
-        ], { :base_url => src })
+        ], { :base_url => src , :username_pattern => /#{HTML::Pipeline::MentionFilter::UsernamePattern.source}/i })
```

Comment:
Let's make this a method as well, if you don't mind! Something like `mention_username_pattern`. Also, is there a way to do this without interpolation? Would love to be able to use the `IGNORECASE` flag instead of relying on this interpolation.